src/SessionManager.php

```diff
@@ -259,6 +288,28 @@ public function getName()
     }
 
     /**
+     * Set session options
+     *
+     * @param array $options
+     * @return SessionManager
+     */
+    public function setOptions(array $options)
```

Comment:
The point of your pull request is to allow having default validators enabled, not to expose arbitrary options. As such, I plan to remove the `setOptions`/`getOptions` methods on merge, as they are not part of that goal, and because the side-effects of options passed to the constructor can be determined programmatically already.